src/browser/browser.zig

```diff
@@ -51,10 +51,9 @@ const log = std.log.scoped(.browser);
 pub const Browser = struct {
     session: *Session,
 
-    pub fn init(alloc: std.mem.Allocator, vm: jsruntime.VM) !Browser {
+    pub fn init(alloc: std.mem.Allocator) !Browser {
         // We want to ensure the caller initialised a VM, but the browser
         // doesn't use it directly...
-        _ = vm;
```

Comment:
Why do you remove the parameter and this line? The comment explain why I'm requesting the parameter even if I don't use it directly.